test/performance/range/views/view_minimiser_hash_benchmark.cpp

```diff
@@ -0,0 +1,118 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/range/views/minimiser_hash.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+#include <seqan3/test/performance/naive_minimiser_hash.hpp>
```

Comment:
sort alphabetically:
```suggestion
#include <seqan3/test/performance/naive_minimiser_hash.hpp>
#include <seqan3/test/performance/sequence_generator.hpp>
```